packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -654,46 +662,36 @@ class MultiSelectComboBox extends ResizeMixin(InputControlMixin(ThemableMixin(El
     }
 
     // Clear all chips except the overflow
-    const chips = Array.from(this._chips).reverse();
-    const overflow = chips.pop();
-
-    chips.forEach((chip) => {
-      chip.remove();
+    Array.from(this._chips).forEach((chip) => {
+      if (chip !== this.$.overflow) {
+        chip.remove();
+      }
     });
 
     const items = [...this.selectedItems];
 
-    let refNode = overflow.nextElementSibling;
-
-    // Use overflow chip to measure min-width
-    const chipMinWidth = this.__getMinWidth(overflow);
-    const inputMinWidth = parseInt(getComputedStyle(this.inputElement).flexBasis);
-    const containerStyle = getComputedStyle(this._inputField);
+    // Detect available max-width for chips
+    const totalWidth = this._inputField.$.wrapper.clientWidth;
+    const inputWidth = parseInt(getComputedStyle(this.inputElement).flexBasis);
 
-    // Detect available width for chips
-    let totalWidth =
-      parseInt(containerStyle.width) -
-      parseInt(containerStyle.paddingLeft) -
-      parseInt(containerStyle.paddingRight) -
-      this.$.toggleButton.clientWidth -
-      inputMinWidth;
+    let maxWidth = totalWidth - inputWidth;
```

Comment:
```suggestion
    let remainingWidth = totalWidth - inputWidth;
```
Note that there are 2 other references to this variable.

Reply:
Done.